core/src/main/java/org/elasticsearch/search/SearchModule.java

```diff
@@ -623,6 +604,17 @@ private void registerBuiltinMovingAverageModels() {
         registerMovingAverageModel(HoltWintersModel.NAME_FIELD, HoltWintersModel::new, HoltWintersModel.PARSER);
     }
 
+    private void registerBuiltinSubFetchPhases() {
+        fetchSubPhases.add(new ExplainFetchSubPhase());
```

Comment:
nitpick: may be better to call `registerFetchSubPhase` instead of adding directly?